app/controllers/volunteer_ops_controller.rb

```diff
@@ -1,7 +1,15 @@
 class VolunteerOpsController < ApplicationController
   layout 'two_columns_with_map'
-  before_filter :authorize, :except => [:show, :index]
+  before_action :authorize, :except => [:search, :show, :index]
 
+  def search
+    @key_words = params[:q]
```

Comment:
maybe @keywords?  or perhaps better call this @query ...? - arguably I think we are on full text search here ...